### DISCUSSION TITLE
ch5/ch5-01

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#159] [General]
# ch5/ch5-01

<The Go Programming Language>中文版

https://gopl-zh.github.io/ch5/ch5-01.html

**Comment by zhangbest5** [comment]
//implemented in assembly language
用汇编语言实现

该说明，直接说明，毕竟中文版，没必要机翻一样

**Comment by Apahno** [comment]
打卡
